### PR TITLE
Nova 4 support

### PR DESCRIPTION
**Pull request by anand-patel**
I have added support for [Nova 4](https://github.com/spatie/nova-tags-field/issues/145), with added dependencies. Also changed code to support Vue 3. 

**Multi tags - Form view** 

Screenshot 2022-04-07 at 11 49 18 AM

**Single tag - Form view** 

Screenshot 2022-04-07 at 11 50 08 AM

**Detail view**
Screenshot 2022-04-07 at 11 49 32 AM

**Index view**
Screenshot 2022-04-07 at 11 50 50 AM

### CODE REVIEW

**Comment by sp4r74cus** [comment]
I can confirm it's working properly.

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     ],
     "require": {
         "php": "^8.0|^7.3",
-        "laravel/nova": "^3.0",
+        "laravel/nova": "^3.0|^4.0",
```
Comment (freekmurze) [inline composer.json:25]: This should be Nova 4 only, right?

Comment (anand-patel) [reply composer.json]: Yes, I have updated the composer.json.

**Comment by freekmurze** [comment]
Thank you!

**Comment by vaites** [comment]
The css file is not present in /dist/ folder, so the load fails:

> Refused to apply style from 'http://localhost/nova-api/styles/nova-tags-field' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

**Pull request merged**